src/ansible_navigator/actions/inventory.py

```diff
@@ -164,9 +164,8 @@ def run(self, interaction: Interaction, app: AppPublic) -> Union[Interaction, No
         """Handle :inventory
 
         :param interaction: The interaction from the user
-        :type interaction: Interaction
-        :param calling_app: The calling_app instance
-        :type calling_app: App
+        :param app: The app instance
+        :return: The pending interaction or none
```

Comment:
How about
```suggestion
        :return: The pending interaction or ``None``
```

Reply:
Or even better:
```suggestion
        :return: The pending interaction or :data:`None`
```

Reply:
The latter will link to https://docs.python.org/3/library/constants.html#None.

Reply:
(there are other places like this — please address them as well)

Reply:
fancy, I like :)